tests/lit-tests/aarch64_dotprod.ispc

```diff
@@ -0,0 +1,30 @@
+// Test checks emitted code for ARM (aarch64) dot product instructions.
```

Comment:
Do we need a test to ensure that udot/sdot instructions are not generated on CPUs that do not support them?

Reply:
Added